src/util/DeepLinkParser.js

```diff
@@ -26,7 +26,19 @@ export function parseDeepLink(uri: string, opts: { aztecoApiKey?: string } = {})
     return parseDownloadLink(url)
   }
 
-  // Handle the edge:// protocol:
+  // Handle bitpay.com links.
+  // We always want to bypass the plugin, even if a scheme (i.e. bitcoin:) is
+  // defined because it is valid for the user to accept any supported currency
+  // besides the specific currency defined in the uri's scheme.
+  if (uri.includes('https:') && uri.includes('bitpay.com')) {
+    if (url.protocol !== 'https:') {
```

Comment:
Is it possible that the uri includes 'https:' but the url protocol wouldn't also be 'https:'? If it is, add examples of acceptable bitpay urls as comments

Reply:
This case is catching a uri scheme representing a coin like "bitcoin:" that would be stored in the url.protocol param.
I can clarify the comment.